core/docker/default/etc/jvm.config

```diff
@@ -12,3 +12,8 @@
 -XX:PerBytecodeRecompilationCutoff=10000
 -Djdk.attach.allowAttachSelf=true
 -Djdk.nio.maxCachedBufferSize=2000000
+-XX:+UnlockDiagnosticVMOptions
+-XX:+UseAESCTRIntrinsics
+# We enable -XX:+UnlockDiagnosticVMOptions and -XX:+UseAESCTRIntrinsics to use a backport introduced by openjdk/jdk11u-dev#410.
```

Comment:
This comment should precede the lines with these options (not go after them)

(same in others)

Reply:
ok, done